html/pfappserver/lib/pfappserver/Base/Form/Role/WithSource.pm

```diff
@@ -14,17 +14,22 @@ use HTML::FormHandler::Moose::Role;
 with 'pfappserver::Base::Form::Role::Help';
 
 use pf::log;
+use pf::constants;
 
 after 'setup' => sub {
     my ($self) = @_;
 
     if($self->for_module->does('captiveportal::Role::MultiSource')){
-        $self->field('source_id')->multiple(1);
-        $self->field('source_id')->options([$self->options_sources(multiple => 1)]);
+        $self->field('multi_source_ids.contains')->options([$self->options_sources(multiple => 1)]);
```

Comment:
That's the way the «framework» for ordered selects works and its hardcoded to required a field called .contains from what I know